collectors/persistentvolumeclaim.go

```diff
@@ -87,9 +98,26 @@ func (collector *persistentVolumeClaimCollector) Collect(ch chan<- prometheus.Me
 	}
 }
 
+// getPersistentVolumeClaimClass returns StorageClassName. If no storage class was
+// requested, it returns "".
+func getPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
+	// Use beta annotation first
+	if class, found := claim.Annotations[v1.BetaStorageClassAnnotation]; found {
```

Comment:
hi, @andyxning 

I had tried to use external package, but failed because:

- [PersistentVolumeClaim from core repo](https://github.com/kubernetes/kubernetes/blob/v1.7.4/pkg/api/v1/helper/helpers.go#L475) use api types under `k8s.io/kubernetes/pkg/api/v1`, not `k8s.io/client-go/pkg/api/v1`.
- k8s.io/client-go `v4.0.0-beta.0` branch only has a [GetPersistentVolumeClaimClass](https://github.com/kubernetes/client-go/blob/v4.0.0-beta.0/pkg/api/helper/helpers.go#L549) function which use `k8s.io/client-go/pkg/api.PersistentVolumeClaim` not `k8s.io/client-go/pkg/api/v1.PersistentVolumeClaim`

So, I found no way to reference an existing package directly. 

IMHO, it's acceptable (though not good) to copy [GetPersistentVolumeClaimClass](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/helper/helpers.go#L561) here because `StorageClass` feature is stable now, and this function has beta annotation backward support. If future releases of `client-go` contains such helper utilities which we can use directly, we can switch to them at that time.